deployment/src/main/java/io/quarkiverse/cxf/deployment/QuarkusCxfProcessor.java

```diff
@@ -1332,6 +1343,14 @@ public void build(
         }
     }
 
+    private String extractJAXBElementParam(FieldInfo f) {
+        if (f.type().name().equals(DotName.createSimple(JAXBElement.class.getName()))) {
+            return f.type().asParameterizedType().arguments().get(0).name().toString();
+        } else {
```

Comment:
I'm doing this with a simple equality and not a hierarchy analysis because type parameters can come and go with each new sub-class and I don't know how to find the right one so I kept it simple.

Of course, that can always be added later.